templates/meilisearch/.platform.template.yaml

```diff
@@ -0,0 +1,24 @@
+version: 1
+
+info:
+  id: platformsh/meilisearch
+  name: Meilisearch
+  description: |
+      <p>This template provides a production-ready demo Meilisearch search engine on Platform.sh. The Meilisearch executable itself is downloaded during the build hook, and the example `movies` index provided in the Meilisearch documentation is added to the engine using the `meilisearch-python` library and Poetry during the deploy hook. You can specifiy the version of Meilisearch by updating the `.platform.app.yaml` file.</p>
```

Comment:
Mh,  if the README we say "It is not production ready, but it can be made so with only a few changes."
So we should probably not say it's production-ready here?